### PR TITLE
Make bios-boot-tutorial.py not rely on prior version of system contracts

### PR DESCRIPTION
**Pull request by linhuang-blockone**
## Change Description

The latest `eosio.bios` requires prior version of system contracts to bootstrap a chain. As a result, `bios-boot-tutorial.py` requires the user to provide old `eosio.contracts-1.8.x` and new `eosio.contracts-1.9.x`. 

This PR utilizes `eosio.boot` contract which provides necessary functions for activation but only relies on `PREACTIVATE_FEATURE`. No old contracts are required.

## Change Type
**Select ONE**
- [x] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: tutorials/bios-boot-tutorial/README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Minimal command line to launch the script below, make sure you replace `EOSIO_CO
 $ cd ~
 $ git clone https://github.com/EOSIO/eos.git
 $ cd ./eos/tutorials/bios-boot-tutorial/
-$ python3 bios-boot-tutorial.py --cleos="cleos" --nodeos=nodeos --keosd=keosd --contracts-dir="/EOSIO_CONTRACTS_DIRECTORY/" -a
+$ python3 bios-boot-tutorial.py --cleos=" --wallet-url http://127.0.0.1:6666 " --nodeos=nodeos --keosd=keosd --contracts-dir="EOSIO_CONTRACTS_DIRECTORY" -w -a
```
Comment (heifner) [inline tutorials/bios-boot-tutorial/README.md:30]: Why non-default wallet location?

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py [hunk 2]
```diff
@@ -293,9 +296,56 @@ def stepCreateTokens():
...
+    retry(args.cleos + 'set contract eosio ' + args.contracts_dir + '/eosio.boot/')
+    sleep(3)
+
+    # activate remaining features
+    # GET_SENDER
+    retry(args.cleos + 'push action eosio activate \'["f0af56d2c5a48d60a4a5b5c903edfb7db3a736a94ed589d0b797df33ff9d3e1d"]\' -p eosio')
+    # FORWARD_SETCODE
+    retry(args.cleos + 'push action eosio activate \'["2652f5f96006294109b3dd0bbde63693f55324af452b799ee137a81a905eed25"]\' -p eosio')
+    # ONLY_BILL_FIRST_AUTHORIZER
+    retry(args.cleos + 'push action eosio activate \'["8ba52fe7a3956c5cd3a656a3174b931d3bb2abb45578befc59f283ecd816a405"]\' -p eosio')
```
Comment (heifner) [inline tutorials/bios-boot-tutorial/bios-boot-tutorial.py:322]: For consistency can we make these all `eosio@active`

File: tutorials/bios-boot-tutorial/README.md (hunk 1 above)
Comment (linhuang-blockone) [reply tutorials/bios-boot-tutorial/README.md:30]: Default wallet should work. Will change.

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py (hunk 2 above)
Comment (linhuang-blockone) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py:322]: Will change. Thanks.

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py [hunk 3]
```diff
@@ -103,6 +103,7 @@ def startNode(nodeIndex, account):
     cmd = (
         args.nodeos +
         '    --max-irreversible-block-age -1'
+        '    --max-transaction-time=1000'
```
Comment (heifner) [inline tutorials/bios-boot-tutorial/bios-boot-tutorial.py:106]: max-transaction-time should never be set larger than block time.

Comment (linhuang-blockone) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py:106]: This was taken from Release-2.0.x. I could not find default block time. Should it be taken out?

Comment (heifner) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py:106]: That should be removed from release-2.0.x then as well. The default of 30 is likely too low hence the argument. I would recommend setting it larger than 30 but less than block time.

Comment (linhuang-blockone) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py:106]: Do you know how much block time is? I cannot find where it is defined.

Comment (linhuang-blockone) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py:106]: Thanks! Was searching for `block_time`. Will set `max-transaction-time` to 200 to be on safe side.

**Review by TimothyBanks — APPROVED** [review]

**Pull request merged**